core/src/test/java/com/devonfw/tools/solicitor/reader/cyclonedx/CyclonedxReaderTests.java

```diff
@@ -49,7 +49,7 @@ public void readMavenFileAndCheckSize() {
     this.cdxr.setInputStreamFactory(new FileInputStreamFactory());
     this.cdxr.setDelegatingPackageURLHandler(this.delegatingPurlHandler);
     this.cdxr.readInventory("maven", "src/test/resources/mavensbom.json", application, UsagePattern.DYNAMIC_LINKING,
-        "cyclonedx", null);
+        "cyclonedx", "cyclonedx", null);
```

Comment:
The packageType within all the other readers is usually null. So, the tests must also succeed with packageType == null.

This applies to all reader tests apart from the CSV reader.